internal/driver/mobile/android.c

```diff
@@ -178,12 +178,12 @@ char* readStream(uintptr_t jni_env, uintptr_t ctx, void* stream, int len, int* t
 		return NULL;
 	}
 
-	char* bytes = malloc(sizeof(char)*count);
+	jbyte* bytes = (jbyte*)malloc(sizeof(char)*count);
```

Comment:
Can you guarantee jbyte and char are the same size, or should the sizeof parameter be updated too?

Reply:
Oversight. Fixed.